snakefmt/parser/syntax.py

```
@@ -275,14 +277,20 @@ class Vocabulary:
     Responsible for recognising snakemake keywords
     """
 
-    spec = dict()
+    spec: "dict[str, Context]" = dict()
```

Ruff (0.15.6)

[warning] Mutable default value for class attribute

(RUF012)